acceptance-tests/features/submit_case.feature

```diff
@@ -1,4 +1,4 @@
-@US07
+@submit_case
```

Comment:
I changed the tags as I was going through as I remember michael made a point about the user story number adding a level of ambiguity to the stories, thinking as we do kinda use both whether I could add two seperate tags and then people could use whichever the find more useful. Will look into it.